metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -8,6 +8,8 @@ performance counters.
 
 You must configure queries for the Windows performance counters that you wish
 to collect. The example below collects processor time and disk writes.
+With `format` you can set the output format for an specific counter. Possible values are
```

Comment:
I don't think it makes sense to expose the intricacies of uint32 vs uint64 because JSON doesn't make any distinction between these types. Maybe just offer `float` and `long` (and `long` uses `uint64` under the hood).